third_party/move/move-vm/transactional-tests/tests/compatibility/entry_private_function.masm

```diff
@@ -0,0 +1,22 @@
+//# publish
+module 0x66::a
+fun foo()
+    ret
+
+
+//# publish
+module 0x66::a
+entry fun foo()
+    ret
+
+
+//# publish
```

Comment:
One convention I started using, whenever there is a publish or run I expect to fail, I do use --verbose (//# publish --verbose). This is echoed and I know whether error is expected or not. Moreover, the error has the VM message printed (I recently removed unstable output from --verbose in tests.)